JDI.Light/JDI.Light/Elements/Composite/MultiDropdown.cs

```diff
@@ -0,0 +1,138 @@
+﻿using JDI.Light.Elements.Base;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
some references are not used:

![image](https://user-images.githubusercontent.com/6689884/54746155-86ebcf00-4bdc-11e9-81ce-38a723f1f6d0.png)